docs/SOPs/Introduction/dataset_wrangling_SOP.md

```diff
@@ -56,21 +52,21 @@ Read more in the relevant [SOP](https://ebi-ait.github.io/hca-ebi-wrangler-centr
 
 ## Gathering data
 
-The first step in the wrangling process is making sure that we have files available to deposit. If no file can be associated with metadata, the submission will be invalid.
+After we are made sure that a DCA is in place, we can start with making sure that we have the files available to deposit. If no file can be associated with metadata, submission will not be possible to be ingested.
```

Comment:
```suggestion
Collecting data files is important because submissions with no data are not allowed in the Data portal, and a submission with no data will fail the validation step when ingested.
```